package.json

```diff
@@ -18,14 +18,15 @@
     "test-manual": "bedrock -f src/test/ts/**/*Test.ts",
     "build": "yarn run clean && tsc -p ./tsconfig.es2015.json && tsc -p ./tsconfig.cjs.json",
     "watch": "tsc -w -p ./tsconfig.es2015.json",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build",
     "deploy-storybook": "yarn storybook-to-ghpages --source-branch=main"
   },
   "keywords": [],
   "author": "Ephox Corporation DBA Tiny Technologies, Inc.",
   "license": "MIT",
   "dependencies": {
+    "@storybook/storybook-deployer": "^2.8.16",
```

Comment:
Was this supposed to be a dependency on the package or a dev dependency?

Reply:
Was _supposed_ to be a devDependency.